migrations/versions/490_add_users_to_briefs.py

```diff
@@ -0,0 +1,32 @@
+"""Add users to briefs
+
+Revision ID: 490
+Revises: 480
+Create Date: 2016-01-25 17:08:03.550222
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '490'
+down_revision = '480'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
```

Comment:
A minor annoyance... can we remove the `commands auto generated by Alembic - please adjust!` comments from the migrations?  Removing these comments feels like saying "I've reviewed Alembic's work and am happy with it".